lib/honeybadger/notice.rb

```diff
@@ -280,6 +280,11 @@ def halted?
       !!@halted
     end
 
+    # Get the parsed exception backtrace.
+    def parsed_backtrace
+      @parsed_backtrace ||= parse_backtrace(backtrace)
```

Comment:
If I understand this correctly, the `parse_backtrace` method will only be called once. With the [javascript implementation](https://github.com/honeybadger-io/honeybadger-js/blob/master/packages/js/src/server.ts#L21) in mind, is there any chance that backtrace is intentionally modified after the call to `parsed_backtrace`?

Reply:
In the Ruby implementation, that processing is done in the `Backtrace.parse` call, using the [filters from the notice class](https://github.com/honeybadger-io/honeybadger-ruby/blob/5646f38b8dafc3d8fa5d0dfaadd91922e8a7bcc2/lib/honeybadger/notice.rb#L157).